micromamba/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ cmake_policy(SET CMP0025 NEW) # Introduced in cmake 3.0
 cmake_policy(SET CMP0077 NEW) # Introduced in cmake 3.13
 project(micromamba)
 
+include(GNUInstallDirs)
+
```

Comment:
`CMAKE_INSTALL_BINDIR` was not always set when building `mamba` and when configuring the `etc/profile.d/mamba.sh` shell script.

Including `GNUInstallDirs` sets `CMAKE_INSTALL_BINDIR` to `bin`.